fire/inspectutils.py

```diff
@@ -66,43 +87,37 @@ def GetArgSpec(fn):
     A named tuple of type inspect.ArgSpec with the following fields:
       args: A list of the argument names accepted by the function.
       varargs: The name of the *varargs argument or None if there isn't one.
-      keywords: The name of the **kwargs argument or None if there isn't one.
+      varkw: The name of the **kwargs argument or None if there isn't one.
       defaults: A tuple of the defaults for the arguments that accept defaults.
+      kwonlyargs: A list of argument names that must be passed with a keyword.
```

Comment:
Fixed the line you linked here. It would probably do to bang on the completion script a little, since the args done changed.